dataland-frontend/tests/component/components/general/CreateApiKeyCard.cy.ts

```diff
@@ -0,0 +1,37 @@
+import CreateApiKeyCard from "@/components/general/apiKey/CreateApiKeyCard.vue";
+import { mount } from "cypress/vue";
+
+describe("Component test for CreateApiKeyCard", () => {
```

Comment:
Please combine 'CreateApiKeyCard.cy.ts` and 'ApiKeyCard.cy.ts` to a new `UserRolesBadgesTest`